### ISSUE TITLE
lyd_parse_mem fails for empty tag

### ISSUE THREAD

**Issue by abhikeshav** [CESNET/libyang#172]
Hi,
Why does `lyd_parse_mem` return NULL pointer when we have a XML like below when using `LYD_OPT_GET` option?
```
<test xmlns="urn:a/>
```
```
module a {
namespace "urn:a";
prefix a;

container test {
  leaf x {
    type string; 
  }
}
```

**Comment by abhikeshav** [comment]
FYI.. also I noticed, using the `LYD_OPT_CONFIG` option for `lyd_parse_mem` for the above XML returns a datanode, which has some other schema that is not the same as `container test`

**Comment by abhikeshav** [comment]
With the latest origin/master, the above is no longer an issue. Thanks a lot!

**Comment by rkrejci** [comment]
it seems that I referenced wrong issue in the commit message - it was solved by bacc744
